utils/create-icons-map.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const path = require('path');
+
+const svgFolder = '../assets/icons/';
+const iconMapFile = '../src/iconsMap.js';
+
+let iconsObject = '';
+
+let iconMapDir = path.dirname(iconMapFile);
+if (!fs.existsSync(iconMapDir)) {
+    fs.mkdirSync(iconMapDir);
+};
+
+
+
+fs.readdirSync(svgFolder).forEach(file => {
+    let fileName = file.split('.')[0];
+    let filePath = svgFolder + file;
+    if (fs.existsSync(filePath)) {
+        const iconBody = fs.readFileSync(filePath).toString();
+        const cleanedIcon = preparIcon(iconBody, fileName);
+        iconsObject += "'" + fileName + '\' : \'' + cleanedIcon + '\', \n';
+    }
+})
+
+const dataToWrite = 'export const Icons = {' + iconsObject + '};';
+
+fs.writeFileSync(iconMapFile, dataToWrite);
+
+console.log('Icons Map created!');
+
+
+function preparIcon(icon, iconName){
+    let iconLine = icon.replace(/\s{2,}/g, '');
+    if(iconLine.match(/(fill=".*")|(id='.*' )|(width=".*")|height=(".*")/g)){
+        console.log('Please, check icon: ', iconName);
+        iconLine =  iconLine.replace(/(fill=".*")|(id='.*')|width=(".*")|height=(".*")/g, '');
+    }
+    iconLine = iconLine.replace('<svg', '<svg class = "svg-icon"')
```

Comment:
1. `.svg-icon` defines icon of size 20x20. What if I want to change the size of the icon?
1. `.svg-icon` includes sub-classes such as `.__angleRight`, `.__chevronDown`. Are they in use?

Reply:
I used the same css which uses ReactApp. I have no requirements for different sizes of Icon.

Reply:
@eliyahen Those are old icons. But that is the correct way to size icons. It definitely should not be in the SVG tag with height/width properties.

Also @fridmaster since you're using the regex twice, you can just create a RegExp object